src/enclave/enclave_init.c

```diff
@@ -8,205 +8,202 @@
-    }
-    app_config->envp[i] = NULL;
+    lkl_mount_disks(&cfg->root, cfg->mounts, cfg->num_mounts, cfg->cwd);
+}
 
-    app_config->cwd = oe_strdup(sgxlkl_config->cwd);
+// In internal OE header openenclave/internal/sgx/eeid_plugin.h
+#define OE_FORMAT_UUID_SGX_EEID_ECDSA_P256                                \
+    {                                                                     \
+        0x17, 0x04, 0x94, 0xa6, 0xab, 0x23, 0x47, 0x98, 0x8c, 0x38, 0x35, \
+            0x1c, 0xb0, 0xb6, 0xaf, 0x0A                                  \
+    }
 
-    return;
-}
+oe_result_t oe_sgx_eeid_attester_initialize(void);
+oe_result_t oe_sgx_eeid_verifier_initialize(void);
 
-static void enclave_get_app_config(sgxlkl_app_config_t* app_config)
+static void get_attestation_evidence()
 {
-    /* Get the application configuration & HDD param from remote server */
-    if (sgxlkl_enclave->mode == HW_RELEASE_MODE)
+    /* Retrieve remote attestation report to exercise Azure DCAP Client
+     * (currently just for testing) */
+    if (sgxlkl_in_hw_debug_mode() || sgxlkl_in_hw_release_mode())
     {
-        sgxlkl_fail("Remote configuration not supported.\n");
-    }
-    else
-    {
-        if (sgxlkl_enclave->app_config_str)
-        {
-            char* err_desc;
-            int ret = parse_sgxlkl_app_config_from_str(
-                sgxlkl_enclave->app_config_str, app_config, &err_desc);
-            if (ret)
-                sgxlkl_fail(
-                    "Failed to parse application configuration: %s\n",
-                    err_desc);
-
-            /* Validate the app config after parsing */
-            ret = validate_sgxlkl_app_config(app_config);
-            if (ret)
-                sgxlkl_fail("Application configuration is not proper\n");
-        }
+        static const oe_uuid_t format_id = {OE_FORMAT_UUID_SGX_EEID_ECDSA_P256};
+
+        oe_sgx_eeid_attester_initialize();
+        oe_sgx_eeid_verifier_initialize();
+
+        size_t evidence_buffer_size = 0;
+        uint8_t* evidence_buffer = NULL;
+        // Endorsements: will work once
+        // https://github.com/openenclave/openenclave/pull/3166 is merged
+        // size_t endorsements_buffer_size = 0;
+        // uint8_t* endorsements_buffer = NULL;
+
+        oe_result_t result = oe_get_evidence(
+            &format_id,
+            NULL,
+            0,
+            NULL,
+            0,
+            &evidence_buffer,
+            &evidence_buffer_size,
+            NULL,
+            0);
+
+        if (result != OE_OK)
+            sgxlkl_fail(
+                "Failed to retrieve attestation evidence: %d.\n", result);
+        else
+            sgxlkl_info("Successfully obtained attestation evidence\n");
+
+        // Note: Since we're using the feature/sgx-lkl-support branch, we can
+        // only verify quotes created from that branch. The verification here
+        // will pass once https://github.com/openenclave/openenclave/pull/3167
```

Comment:
No, it's not merged into the hash the submodule points to. Depends on #505. I've removed this example completely though, as it will have to look different because of the layering.